api-common-java/src/test/java/com/google/api/core/SettableApiFutureTest.java

```diff
@@ -57,15 +57,19 @@ public void testCancel() throws Exception {
     Truth.assertThat(future.isCancelled()).isTrue();
   }
 
-  @Test(expected = ExecutionException.class)
-  public void testException() throws Exception {
-    SettableApiFuture<Integer> future = SettableApiFuture.<Integer>create();
-    future.setException(new Exception());
-    future.get();
+  @Test
+  void testException() {
+    Assertions.assertThrows(
+        Exception.class,
```

Comment:
Hmm, odd. The old test had `@Test(expected = ExecutionException.class)`. I wonder what changed since it looks like it's throwing Exception back now.

Reply:
I think since `ExecutionException` is a subtype of `Exception`, both are technically correct (I tested both and they both pass). But to your point of keeping it the same as the previous one, I've updated to have it `assertThrows(ExecutionException.class)`